packages/example-apps/basic/src/utils/fn-helper.ts

```diff
@@ -0,0 +1,9 @@
+import { helper } from './helper-with-services';
+
+
+export const fn = helper(function([method, ...args]) {
+  return  (...params)  => {
+      let el = [].concat(args, params);
+      return method(...el);
+  }
+});
```

Comment:
```suggestion
export function fn([method, ...args]) {
  return  (...params)  => {
      let el = [].concat(args, params);
      return method(...el);
  }
};
```

Reply:
It would actually be:

```js
export function fn(method, ...args) {
  return  (...params)  => {
    let el = [].concat(args, params);
    return method(...el);
  }
}
```

The outer function only receives positional parameters